docsite/rst/playbooks_filters.rst

```diff
@@ -300,13 +318,17 @@ Hash types available depend on the master system running ansible,
 Other Useful Filters
 --------------------
 
+To add quotes for shell usage::
+
+    - shell: echo={{ some_string_value | quote }} 
+
 To use one value on true and another on false (since 1.9)::
 
-   {{ (name == "John") | ternary('Mr','Ms') }}
+    {{ (name == "John") | ternary('Mr','Ms') }}
```

Comment:
most of the code indents were 4 spaces, some were 3.  can remove if not necessary/desired.